src/phase1.hpp

```diff
@@ -520,7 +517,7 @@ void* phase1_thread(THREADDATA* ptd)
         Sem::Post(ptd->mine);
     }
 
-    return 0;
```

Comment:
as far as I can tell, this is the only return statement in this function. This function doesn't *have* to return anything. I think it would be better to define the function to not return anything and just remove the return statement